### PR TITLE
Docs md and pdf

### PR DESCRIPTION
**Pull request by konrad-armatys**

### CODE REVIEW

File: .vscode/settings.json
Comment (barszu) [inline .vscode/settings.json:1]: ten plik to chb nie jest potrzebny

Comment (karmatys8) [reply .vscode/settings.json:1]: Jak masz code spell to Ci podkreśla te słowa co jest strasznie irytujące.
Ale jak chcesz to mogę usunąć.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# 🛒 Sklep Internetowy - Projekt MERN 🛒
+
+## Skład grupy 👨‍💻 👨‍💻 👨‍💻
+(imiona i nazwiska, adresy email):
+
+- **Bartłomiej Szubiak** bszubiak@student.agh.edu.pl
+- **Szymon Kubiczek** kubiczek@student.agh.edu.pl
+- **Konrad Armaty**s karmatys@student.agh.edu.pl
```
Comment (karmatys8) [inline README.md:8]: ;s; is not bold

File: docs/README.md
Comment (karmatys8) [inline docs/README.md:1]: Now that we have models included in docs itself should we keep them as separate files as well?

Imo docs/backend/models should be deleted.

**Review by konrad-armatys — COMMENTED** [review]
There are some changes that imo should be made.
Once we decide how exactly to finish those docs I can make those changes.

Also should I include generated pdf on the GitHub?
I am asking because you deleted the previous version.

**Pull request merged**